### PR TITLE
correct instructions for enabling jenkins autoprovision

### PR DESCRIPTION
**Pull request by bparees**
depends on https://github.com/openshift/origin/pull/10612

### CODE REVIEW

**Comment by bparees** [comment]
@openshift/team-documentation ptal

**Comment by vikram-redhat** [comment]
Don't think it requires a peer review. Marked it for future 3.3 release. When https://github.com/openshift/origin/pull/10612 merges, we can merge this one into master.

File: install_config/configuring_pipeline_execution.adoc [hunk 1]
```diff
@@ -40,22 +40,21 @@ To modify the content of the default template:
 $ oc edit template jenkins -n openshift
 ----
 
-To use a different template:
+To use a different template, add the following to your master config file:
```
Comment (ahardin-rh) [inline install_config/configuring_pipeline_execution.adoc]: s/config/configuration

**Comment by ahardin-rh** [comment]
@bparees just one comment from me. Thanks!

**Comment by bparees** [comment]
@ahardin-rh done.

**Comment by bparees** [comment]
@ahardin-rh @vikram-redhat pr has merged.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/configuring_pipeline_execution.adoc#install-config-configuring-pipeline-execution[Configuring Pipeline Execution]
|Corrected instructions for enabling Jenkins auto-provision.
%

**Pull request merged**